src/main/java/com/rarchives/ripme/ripper/AbstractHTMLRipper.java

```diff
@@ -53,6 +53,7 @@ public URL sanitizeURL(URL url) throws MalformedURLException {
     protected boolean hasDescriptionSupport() {
         return false;
     }
+    public boolean hasASAPRipping() { return false; }
```

Comment:
Should this go into AbstractRipper instead? I imagine AbstractJSONRipper (and future RSS) could use this functionality as well.

Reply:
> Should this go into AbstractRipper instead?

Now that I think about it, yes

Reply:
@metaprime done in my last push